src/LiveComponent/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # CHANGELOG
 
+-   Use container.build_hash instead of kernel.secret in CacheWarmer
```

Comment:
I don't see this change as requiring a changelog entry, to me it's a "minor" change, not a new "feature", since it doesn't have any user-visible side effect except DX-related things (aka you cannot "build on it", while you can build on a "feature")

But if we want to keep it, I guess this needs a version:
```suggestion
## 2.23.0

-   Use container.build_hash instead of kernel.secret in CacheWarmer
```

Reply:
OK, feel free to edit this to your liking or reject the last commit.